scheduler/scheduler.go

```diff
@@ -366,9 +395,14 @@ func containerCapacity(numCurrentParents, numCurrentContainers, numContainersToS
 // createParents creates host intent documents for each parent
 func createParents(parent distro.Distro, numNewParents int, pool *evergreen.ContainerPool) []host.Host {
 	hostsSpawned := make([]host.Host, numNewParents)
-
+	options, err := generateParentHostOptions(parent.ProviderSettings, pool)
```

Comment:
I don't think this should be needed - outside of maintaining a list of already downloaded images, parents should not know specifics of their child containers. We should save this data in the child and then do the right thing when it's time to start the container

Reply:
i was briefly confused about parents but yeah actually nothing here needs to be changed.